pulp_container/app/models.py

```diff
@@ -91,6 +93,8 @@ class Manifest(Content):
             field is used only for a manifest-list type Manifests.
     """
 
+    protected_from_reclaim = True
```

Comment:
I would introduce this class constant defaulting to to True in pulpcore and only overwrite if reclaim is supported.
This way plugins can opt in to this feature, and don't need to opt out.
Also it's supposed to be a constant, so it should be all caps.

Reply:
it is set to true in core, it is just mentioned here in a redundant way so i will remove it